libs/ui-lib-tests/cypress/views/customManifestsPage.ts

```diff
@@ -0,0 +1,35 @@
+export const customManifestsPage = {
+  getFileName: () => {
+    return cy.findByTestId('filename-0');
+  },
+  getFileNameError: () => {
+    return cy.get('div.pf-c-form__helper-text.pf-m-error');
+  },
+  getStartFromScratch: () => {
+    return cy.get('.pf-c-file-upload .pf-c-empty-state__secondary > button');
+  },
+  fileUpload: () => {
+    return cy.get('input[type="file"]');
+  },
+  getYamlContentError: () => {
+    return cy.get('div.pf-c-form__helper-text.pf-m-error');
+  },
+  getAlertTitle: () => {
+    return cy.get('div.pf-c-alert.pf-m-danger > h4.pf-c-alert__title');
+  },
+  getLinkToAdd: () => {
+    return cy.findByTestId('add-manifest');
+  },
+  getManifest1Id: () => {
+    return cy.findByTestId('filename-1');
+  },
+  lastFileUpload: () => {
+    return cy.get('input[type="file"]:last');
+  },
+  getRemoveManifestButton: () => {
+    return cy.get('[aria-label="remove manifest"]:last');
+  },
```

Comment:
Same as with the file upload - use an index and target the button in a manifest specified by the index.